local-proxy/src/server.ts

```diff
@@ -35,7 +36,9 @@ const whitelistedModules = new Map(
   }).filter((pair): pair is [string, any] => pair !== undefined)
 );
 
-const app = express();
+const app = new Koa();
+app.use(bodyParser({ enableTypes: ['json'], strict: false, detectJSON: () => true }));
```

Comment:
I think no need for `detectJSON`, as all requests are generated by browser and no need for manual testing with cURL where omitting content-type header makes it easier.

We can also assume `strict: true`, since the requests are always sending objects - but this seems less critical since we verify the parameters anyway.